wazo_dird/database/queries/phonebook.py

```diff
@@ -11,54 +16,97 @@
     NoSuchPhonebook,
 )
 
-from .base import BaseDAO, compute_contact_hash, list_contacts_by_uuid
-from .. import Contact, ContactFields, Phonebook, Tenant
+from .base import BaseDAO, ContactInfo, compute_contact_hash, list_contacts_by_uuid
+from .. import Contact, ContactFields, Phonebook
+
+
+class PhonebookKey(TypedDict, total=False):
+    id: int
+    uuid: str
+
+
+class PhonebookDict(TypedDict):
+    id: int
+    uuid: str
+    description: str
+    name: str
+    tenant_uuid: str
+
+
+def phonebook_key_to_predicate(phonebook_key: PhonebookKey) -> ColumnElement:
+    assert 'id' in phonebook_key or 'uuid' in phonebook_key
```

Comment:
This is like a runnable comment about the expectation of `PhonebookKey`, to complete information given by the type,  since it's not possible in python 3.9 to specify at the type level that a `PhonebookKey` MUST contain one of `id` or `uuid`(either it must contain both or may contain either or none at all).

Reply:
See with @DrPyser . Since this assert is not part of logic and act more as a comment code, it's fine